Automattic/woocommerce-connect-client
# Add connectforwoocommerce text domain to all strings and load it

Fixes #474

To test:
- Delete `i18n/strings.php`
- `npm run i18n`
- Make sure `i18n/strings.php` gets regenerated with all the strings
- Open each of checkout, settings, self-help and a shipping method instance and make sure strings appear and no errors are logged in php errors nor the javascript console
